### PR TITLE
Adds ekatana to traitor gamemode and makes it bulky.

### PR DESCRIPTION
**Pull request by Swagile**
:cl: Swagile
add: Adds energy katana to traitor uplinks for a immense price alongside making it heavy. Buy at your own risk.
/:cl:

Adds energy katana to the standard traitor gamemode.

Allows a controlled blink, does a bit more damage than esword, and can emag. In return, it has been nerfed so now the only place you can hold it is your belt. It can no longer be backpacked. Buy it at your own risk as its a FULL LOUD weapon.

also i have no fucking idea if i did this modular thing right or not, so here goes nothing ???

### CODE REVIEW

File: hippiestation/code/modules/ninja/energy_katana.dm [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	desc = "A katana infused with strong energy."
+	icon_state = "energy_katana"
+	item_state = "energy_katana"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	force = 40
+	throwforce = 20
+	block_chance = 50
+	armour_penetration = 50
+	w_class = WEIGHT_CLASS_BULKY
```
Comment (KayeArray) [inline hippiestation/code/modules/ninja/energy_katana.dm]: the only line you need in this file is this

File: hippiestation/code/modules/ninja/energy_katana.dm [hunk 2]
```diff
@@ -0,0 +1,103 @@
+/obj/item/energy_katana
```
Comment (KayeArray) [inline hippiestation/code/modules/ninja/energy_katana.dm:1]: and this

**Comment by PopNotes** [comment]
still no

File: hippiestation/code/modules/ninja/energy_katana.dm (hunk 2 above)
Comment (Swagile) [reply hippiestation/code/modules/ninja/energy_katana.dm]: i added this on my side, committed, and pushed it but its not showing on github

weird???

Comment (Spacedong) [reply hippiestation/code/modules/ninja/energy_katana.dm]: are you sure you added it, saved it, closed out, then commited on the branch, then synced?

**Comment by ShadowDeath6** [comment]
>bit more damage

how much damage does the esword do? this does 40

File: hippiestation/code/modules/ninja/energy_katana.dm (hunk 2 above)
Comment (Swagile) [reply hippiestation/code/modules/ninja/energy_katana.dm]: i saved it, committed to branch, then pulled

**Comment by Swagile** [comment]
the esword does 30

this does 40

File: hippiestation/code/modules/ninja/energy_katana.dm [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
```
Comment (ghost) [inline hippiestation/code/modules/ninja/energy_katana.dm:1]: add this to the dme

Comment (ghost) [inline hippiestation/code/modules/ninja/energy_katana.dm:1]: *NO* Make this a subtype, don't alter the original katana

**Comment by Dudeinyourbackyard** [comment]
once its all ironed out, it could be good to see in play
maybe substitute out the controlled blink for something else but either way +1

**Comment by MoreRobustThanYou** [comment]
Code is shitty. Make it a subtype of the normal e-katana and reopen.

**Pull request closed without merging**